extensions/amp-3q-player/0.1/test/test-amp-3q-player.js

```diff
@@ -0,0 +1,54 @@
+/**
```

Comment:
@aghassemi It's still not working, this.iframe_.contentWindow is still not present. The test is not passing.